src/components/ArticlePage/MarkdownHtml.js

```diff
@@ -89,8 +89,15 @@ const Html = styled.div`
   }
   h2 {
     ${H4.styles}
-    margin: ${2 * baselineSpacing}px 0 ${baselineSpacing}px 0;
+    margin: ${1.5 * baselineSpacing}px 0 ${baselineSpacing}px 0;
     max-width: ${props => props.theme.contentMaxWidth}px;
+    border-top: 1px solid #CCCCCC;
+    padding-top: 29px;
+
+    @media (min-width: ${baselineBreakpoint}px) {
+      margin: ${2 * baselineSpacing - 2}px 0 ${baselineSpacing}px 0;
```

Comment:
Maybe use a multiple of `baselineLarge` here for clarity as well. But where does the `2px` come from? The border width is `1px`, or is the padding value affecting this?

Reply:
You should also adjust the `first-child` top margin, now the baseline breaks at least in the Flex intro article. The height of the `h2` is not a multiple of the baseline.

![Screen Shot 2019-04-08 at 16 14 23](https://user-images.githubusercontent.com/53923/55727698-35847200-5a1b-11e9-9aa1-f8773cee3722.png)

Reply:
hmm, this is a tricky one because some of the articles don't have a paragraph between ingress and H2, but most of them do.